app/services/explorer.js

```diff
@@ -655,44 +655,41 @@ export default Ember.Service.extend({
      * @param {DS.Store} store
      * @return {Ember.RSVP.Promise<BucketList>} Result of the AJAX request
      */
-    getBucketList(cluster, bucketType, store) {
-        console.log('Refreshing buckets for bucketType: %O', bucketType);
-        var clusterId = cluster.get('clusterId');
-        var bucketTypeId = bucketType.get('bucketTypeId');
-        var url = this.apiURL + 'explore/clusters/' + clusterId +
-            '/bucket_types/' + bucketTypeId + '/buckets' ;
-        var explorer = this;
+    getBucketList(cluster, bucketType, store, start=1, rows=1) {
+        let explorer = this;
+        let clusterId = cluster.get('clusterId');
+        let bucketTypeId = bucketType.get('bucketTypeId');
+        let url = `${this.apiURL}explore/clusters/${clusterId}/bucket_types/${bucketTypeId}/buckets?start=${start}&rows=${rows}`;
```

Comment:
FYI, I cleaned this up to use [ES6 string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings)

I set the `start` and `rows` defaults manually ([another ES6 feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters):smile:), but I am curious where you think these should live? `config/environment` doesn't make sense. Maybe here is fine?